nbs/00_utils.ipynb

```diff
@@ -12,17 +12,27 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.        if not kwargs:
</pre><p>This if statement is redundent. If nothing is passed, we can simply use the default value.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BirkhoffG/jax-relax/pull/20/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Reply:
Change Implemented